CHANGELOG.md

```diff
@@ -4,6 +4,12 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## 0.2.6-M1
+### Added
+
+### Fixed
+- Update Dependencies.md 
```

Comment:
Will there be another PR with the updated `DEPENDENCIES` file? BTW, the file is named without _.md_.

Reply:
Yes, there will be an other PR for updating the dependencies.